### PR TITLE
Edited Spectral lander for consistency, clarity

### PR DESCRIPTION
**Pull request by adamd**

### CODE REVIEW

File: netlify/landings/spectral.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ featureSection:
         in turn makes the APIs better.
       image: /images/mocking_instant.png
       shortName: Improve API Quality
```
Comment (adamd) [inline netlify/landings/spectral.yaml]: Other landers tend to make the shortName a noun. For example: API Quality

File: netlify/landings/spectral.yaml [hunk 2]
```diff
@@ -49,7 +49,7 @@ featureSection:
         ruleset updates automatically.
       image: /images/mocking_virtualization.png
       shortName: Reduce Decision Making
```
Comment (adamd) [inline netlify/landings/spectral.yaml]: If we remove the verb here, too, I'd suggest changing up the shortName to: API Consistency

File: netlify/landings/spectral.yaml [hunk 3]
```diff
@@ -16,13 +16,13 @@ featureSection:
...
     enabled: true
-    text: Do you want to get feedback on your API descriptions right now?
+    text: Do you want feedback on your API descriptions right now?
 
   title: Your API descriptions deserve better than "Technically Correct"
   description: >-
     Spectral is an open source OpenAPI-linter, which can tell you not just if
     your API descriptions are valid, but if they are any good. Use our baked-in
-    OpenAPI Specification v2/v3 rulesets (formerly known as Swagger specification), 
+    rulesets for OpenAPI Specification v2/v3 (formerly known as Swagger specification),
```
Comment (philsturgeon) [inline netlify/landings/spectral.yaml]:
```suggestion
    rulesets for OpenAPI v2/v3 (formerly known as Swagger specification), 
```

File: netlify/landings/spectral.yaml (hunk 1 above)
Comment (philsturgeon) [reply netlify/landings/spectral.yaml]: Cool ok!

Comment (XVincentX) [inline netlify/landings/spectral.yaml]:
```suggestion
      shortName: API Quality
```

**Pull request merged**